mne/preprocessing/infomax_.py

```diff
@@ -292,7 +292,8 @@ def infomax(data, weights=None, l_rate=None, block=None, w_change=1e-12,
             # for extended Infomax
             if extended:
                 signs = np.identity(n_features)
-                signs.flat[slice(0, n_features * n_subgauss, n_features)]
+                # XXX ?
+                # signs.flat[slice(0, n_features * n_subgauss, n_features)]
```

Comment:
...but @dengemann here the code passes regardless of whether I leave this line commented, or do `signs = signs.flat[...]`. It's because these lines of code are currently untested. In order to get these lines hit, somehow I need to get into this conditional with a `wts_blowup`, do you know how to do it?